internal/experiment/generation/transformer.go

```diff
@@ -135,12 +135,13 @@ func (t *Transformer) Transform(nodes []*yaml.RNode, selected []interface{}) ([]
 	}
 
 	// Serialize the experiment as a YAML node
-	if expNode, err := (sfio.ObjectSlice{&exp}).Read(); err != nil {
+	expNode, err := (sfio.ObjectSlice{&exp}).Read()
+	if err != nil {
 		return nil, err
-	} else {
-		result = append(expNode, result...) // Put the experiment at the front
 	}
 
+	result = append(expNode, result...) // Put the experiment at the front
+
```

Comment:
If you're going to break up the scoping-if, at least give the poor comment a promotion to its own line ;)